app-admin/flannel/files/flanneld.service

```diff
@@ -13,12 +13,13 @@ Environment="TMPDIR=/var/tmp/"
 Environment="DOCKER_HOST=unix:///var/run/early-docker.sock"
 Environment="FLANNEL_VER={{flannel_ver}}"
 Environment="ETCD_SSL_DIR=/etc/ssl/etcd"
+Environment="FLANNEL_ENV_FILE=/run/flannel/options.env"
 LimitNOFILE=40000
 LimitNPROC=1048576
 ExecStartPre=/sbin/modprobe ip_tables
 ExecStartPre=/usr/bin/mkdir -p /run/flannel
 ExecStartPre=/usr/bin/mkdir -p ${ETCD_SSL_DIR}
-ExecStartPre=/usr/bin/touch /run/flannel/options.env
+ExecStartPre=-/usr/bin/touch ${FLANNEL_ENV_FILE}
```

Comment:
My thinking was if changed from default (/run/flannel/options.env) the path may not exist and touch would fail. Figured it would be better that docker errors with 'file doesn't exist', rather than the touch command failing. But the read-only case is valid too.